viz-lib/src/lib/utils.js

```diff
@@ -41,3 +41,16 @@ export function formatColumnValue(value, columnType = null) {
 
   return value;
 }
+
+/**
+ * Gets the element length at closest (ceil) position from the nth percentile in a list
+ * @param {Array|Object} list
+ * @param {number} percentile Closest hundreth percentile
+ * @param {string | string[]} sortIteratee Lodash's iteratee or iteratee list
+ */
+export function getItemOfPercentileLength(list, percentile, sortIteratee = "length") {
```

Comment:
No need to put it in the `viz-lib` utils as there are no visualizations using it.